names/database.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+from typing import Optional
+
+from sqlalchemy import Column, Integer, String
+
+from pie.database import database, session
+
+
+class Price(database.base):
+    """Prices for nickname subcommands"""
+
+    __tablename__ = "fun_names_prices"
+
+    command_name = Column(String, primary_key=True)
+    price = Column(Integer, default=0)
```

Comment:
Given the fact that we know we'll only have two commands, the table columns can be `guild_id`, `set_price`, `unset_price`.

Reply:
After that, the functions can be
```py
@classmethod
def set(cls, guild_id: int, set_price: int, reset_price: int) -> Price:
    """Add or update prices."""
    ...

@classmethod
def get(cls, guild_id: int) -> Price:
   """Get prices in supplied guild."""
    ...

@classmethod
def remove(cls, guild_id: int) -> bool:
    """Remove prices in supplied guild."""
    ...
```

While your solution would work, my example follows the conventions in the codebase.